### ISSUE TITLE
Migrate cluster policy storage is broken

### ISSUE THREAD

**Issue by deads2k** [openshift/origin#14743]
https://github.com/openshift/openshift-ansible/pull/4476 exposed this, but merged anyway.

**Comment by mfojtik** [comment]
bumping to P0 as this is now merge/test queue blocker. @smarterclayton FYI

**Comment by mfojtik** [comment]
@smarterclayton is somebody working on this? i see that last change to migrate storage was you :-)  This is failing for every single test :-(

@sdodson I thought the revert will fix this.

**Comment by mfojtik** [comment]
also the message changed slightly:

```
Output to stdout: summary: total=0 errors=0 ignored=0 unchanged=0 migrated=0 error: exited without processing all resources: Get https://ip-172-18-1-109.ec2.internal:8443/apis/batch/v1/jobs: dial tcp 172.18.1.109:8443: getsockopt: connection refused Output to stderr: error: exit directly
```

seems like the server died?

**Comment by smarterclayton** [comment]
@enj is working on this, yes

**Comment by smarterclayton** [comment]
Check the logs generated by your start - last time it was the PR, not
this code.  This just happens to be the first thing that runs after
the master starts.

**Comment by mfojtik** [comment]
that one seems to be me:

```
Error starting "openshift.io/sdn" (User "system:serviceaccount:openshift-infra:sdn-controller" cannot get clusternetworks at the cluster scope)
```

fun.

**Comment by enj** [comment]
Fixed by #14764

**Comment by deads2k** [comment]
@sdodson the storage update should be fixed now.
